torch/_torch_docs.py

```diff
@@ -8329,38 +8334,34 @@ def merge_dicts(*dicts):
                tensors for `U` and `Vh`, whereas :func:`torch.linalg.svd` returns
                empty tensors.
 
-.. note:: The singular values are returned in descending order. If :attr:`input` is a batch of matrices,
-          then the singular values of each matrix in the batch are returned in descending order.
-
 .. note:: The `S` tensor can only be used to compute gradients if :attr:`compute_uv` is `True`.
 
 .. note:: When :attr:`some` is `False`, the gradients on `U[..., :, min(m, n):]`
           and `V[..., :, min(m, n):]` will be ignored in the backward pass, as those vectors
           can be arbitrary bases of the corresponding subspaces.
 
-.. note:: The implementation of :func:`torch.linalg.svd` on CPU uses LAPACK's routine `?gesdd`
-          (a divide-and-conquer algorithm) instead of `?gesvd` for speed. Analogously,
+.. note:: The implementation of :func:`torch.linalg.svd` on CPU uses LAPACK's routine `gesdd`
+          (a divide-and-conquer algorithm) instead of `gesvd` for speed. Analogously,
           on GPU, it uses cuSOLVER's routines `gesvdj` and `gesvdjBatched` on CUDA 10.1.243
           and later, and MAGMA's routine `gesdd` on earlier versions of CUDA.
 
 .. note:: The returned `U` will not be contiguous. The matrix (or batch of matrices) will
           be represented as a column-major matrix (i.e. Fortran-contiguous).
 
-.. warning:: The gradients with respect to `U` and `V` will only be finite when the input does not
-             have zero nor repeated singular values.
-
-.. warning:: If the distance between any two singular values is close to zero, the gradients with respect to
-             `U` and `V` will be numerically unstable, as they depends on
-             :math:`\frac{1}{\min_{i \neq j} \sigma_i^2 - \sigma_j^2}`. The same happens when the matrix
-             has small singular values, as these gradients also depend on `S⁻¹`.
-
-.. warning:: For complex-valued :attr:`input` the singular value decomposition is not unique,
-             as `U` and `V` may be multiplied by an arbitrary phase factor :math:`e^{i \phi}` on every column.
+.. warning:: The singular values of a matrix are not unique. Any pair of left and right singular values may
+             be multiplied by `-1` in the real case or by a factor of norm `1` :math:`e^{i \phi}` in the complex case.
```

Comment:
How would these elements be paired? Is it corresponding columns in the U and V matrices? 

"in the real case" -> "when U and V are real-valued" 
"in the complex case" -> "when U and V are complex"

I'm not sure I understand the "factor of norm 1 e^(i * phi)" scaling?

Reply:
What about making this more explicit:
![image](https://user-images.githubusercontent.com/3291265/115227618-98c83500-a108-11eb-94ff-85b8310036f6.png)

Compare this with the proposal from https://github.com/pytorch/pytorch/pull/56265#discussion_r615752325

Reply:
This is much clearer and really excellent. I would consider making it a note, however, since "warning" should probably be reserved for when the user is expected to take an action.